doc/source/overview/getting_started.md

```diff
@@ -165,4 +165,4 @@ preds = od.predict(X_test)
 The predictions are returned in a dictionary with as keys `meta` and `data`. `meta` contains the detector's metadata while `data` is in itself a dictionary with the actual predictions. It has either `is_outlier`, `is_adversarial` or `is_drift` (filled with 0's and 1's) as well as optional `instance_score`, `feature_score` or `p_value` as keys with numpy arrays as values.
```

Comment:
Mention that other relevant values can also be returned in the `data` dict as values.